README.md

```diff
@@ -5,70 +5,102 @@ author: jmprieur
+### Step 2: Create the sample
 
-### Step 2:  Register the sample with your Azure Active Directory tenant
+This sample was created from the 2.0 [dotnet new mvc](https://docs.microsoft.com/dotnet/core/tools/dotnet-new?tabs=netcore2x) template with `SingleOrg` authentication. You can create the sample from the command line or clone/download this repository:
 
-1. Sign in to the [Azure portal](https://portal.azure.com).
-2. On the top bar, click on your account and under the **Directory** list, choose the Active Directory tenant where you wish to register your application.
-3. Click on **More Services** in the left hand nav, and choose **Azure Active Directory**.
-4. Click on **App registrations** and choose **Add**.
-5. Enter a friendly name for the application, for example 'WebApp-OpenIDConnect-DotNet' and select 'Web Application and/or Web API' as the Application Type. For the sign-on URL, enter the redirect URI for the sample, which is by default `https://localhost:44353/signin-oidc`. Click on **Create** to create the application.
-6. While still in the Azure portal, choose your application, click on **Settings** and choose **Properties**.
-7. Find the Application ID value and copy it to the clipboard.
-8. In the same page, change the `logoutUrl` property to `https://localhost:44353/Account/EndSession`.  This is the default single sign out URL for this sample.
-9. For the App ID URI, enter `https://<your_tenant_name>/WebApp-OpenIDConnect-DotNet`, replacing `<your_tenant_name>` with the name of your Azure AD tenant. 
+- To create the sample from the command line, execute the following command:
+
+  ```console
+  dotnet new mvc --auth SingleOrg --client-id <CLIENT_ID_(APP_ID)> --tenant-id <TENANT_ID> --domain <TENANT_DOMAIN>
+  ```
+  Use the values that you recorded from the Azure portal for \<CLIENT\_ID\_(APP\_ID)>, \<TENANT\_ID>, and \<TENANT\_DOMAIN>.
+
+- To clone/download this sample, execute the following command from your shell or command line:
+
+  ```console
+  git clone https://github.com/Azure-Samples/active-directory-dotnet-webapp-openidconnect-aspnetcore.git
+  ```
 
-### Step 3:  Configure the sample to use your Azure Active Directory tenant
+  In the **appsettings.json* file, provide values for the `Domain`, `TenantId`, and `ClientID` that you recorded earlier from the Azure portal.
 
-1. Open the solution in Visual Studio 2017.
-2. Open the `config.json` file.
-3. Find the `Tenant` property and replace the value with your AAD tenant name.
-4. Find the `ClientId` and replace the value with the Application ID from the Azure portal.
-5. If you changed the base URL of the sample, find the app key `ida:PostLogoutRedirectUri` and replace the value with the new base URL of the sample.
+If you aren't using [Visual Studio](https://www.visualstudio.com/vs/) as your development environment and created the project with the `dotnet new mvc` command:
 
-### Step 4:  Run the sample
+- Add Bower to the project file:
```

Comment:
Why do we need to do that? I did not (I did Dotnet run immediately) and that worked?